components/src/organisms/CommandText/useCommandTextString/utils/commandText/getFlexStackerCommandText.ts

```diff
@@ -15,18 +17,152 @@ export type FlexStackerCommand =
   | FlexStackerFillRunTimeCommand
   | FlexStackerEmptyRunTimeCommand
 
+export const KEYS_BY_COMMAND_TYPE: {
+  [commandType in FlexStackerCommand['commandType']]: string
+} = {
+  'flexStacker/retrieve': 'flex_stacker_retrieve',
```

Comment:
We'll need to add branded to all of these if they're going to be used also!
```suggestion
  'flexStacker/retrieve': 'branded:flex_stacker_retrieve',
```

Reply:
added branded when rendering the string